packages/opentelemetry-node-tracer/test/NodeTracer.test.ts

```diff
@@ -27,24 +27,42 @@ import { AsyncHooksScopeManager } from '@opentelemetry/scope-async-hooks';
 import { NodeTracer } from '../src/NodeTracer';
 import { TraceOptions } from '@opentelemetry/types';
 import { Span } from '@opentelemetry/basic-tracer';
+import * as path from 'path';
 
 const sleep = (time: number) =>
   new Promise(resolve => {
     return setTimeout(resolve, time);
   });
 
+const INSTALLED_PLUGINS_PATH = path.join(
+  __dirname,
+  'instrumentation',
+  'node_modules'
+);
+
 describe('NodeTracer', () => {
+  before(() => {
+    module.paths.push(INSTALLED_PLUGINS_PATH);
+  });
+
+  afterEach(() => {
+    // clear require cache
+    Object.keys(require.cache).forEach(key => delete require.cache[key]);
```

Comment:
Does this "un-require" everything? What would be in the cache at this point? Since (I think?) only 1 test is requiring in your test packages.

Reply:
Yes, it will clean the everything from the cache. Do you suggest to delete a particular module from the cache instead of everything?